tests/charts_app.py

```diff
@@ -0,0 +1,365 @@
+from random import randint
+import json
+
+import dash_mantine_components as dmc
+
+
+_dash_renderer._set_react_version("18.2.0")
+
+data = [
+    {"month": "January", "Smartphones": 1200, "Laptops": 900, "Tablets": 200},
+    {"month": "February", "Smartphones": 1900, "Laptops": 1200, "Tablets": 400},
+    {"month": "March", "Smartphones": 400, "Laptops": 1000, "Tablets": 200},
+    {"month": "April", "Smartphones": 1000, "Laptops": 200, "Tablets": 800},
+    {"month": "May", "Smartphones": 800, "Laptops": 1400, "Tablets": 1200},
+    {"month": "June", "Smartphones": 750, "Laptops": 600, "Tablets": 1000}
+]
+
+bubble_data = [
+  { "hour": '08:00', "index": 1, "value": 150 },
+  { "hour": '10:00', "index": 1, "value": 166 },
+  { "hour": '12:00', "index": 1, "value": 170 },
+  { "hour": '14:00', "index": 1, "value": 150 },
+  { "hour": '16:00', "index": 1, "value": 200 },
+  { "hour": '18:00', "index": 1, "value": 400 },
+  { "hour": '20:00', "index": 1, "value": 100 },
+  { "hour": '22:00', "index": 1, "value": 160 },
+]
+
+bubbleChart = dmc.BubbleChart(
+    h=60,
+    id={"type": "chart", "index": "bubble"},
+    data=bubble_data,
+    range=[16, 225],
+    label="Sales/hour",
+    color="lime.6",
+    dataKey={ "x": 'hour', "y": 'index', "z": 'value' },
+)
+
+barchart_data = [
+    {"date": "Mar 22", "Apples": 2890, "Oranges": 2338, "Tomatoes": 2452},
+    {"date": "Mar 23", "Apples": 2756, "Oranges": 2103, "Tomatoes": 2402},
+    {"date": "Mar 24", "Apples": 3322, "Oranges": 986, "Tomatoes": 1821},
+    {"date": "Mar 25", "Apples": 3470, "Oranges": 2108, "Tomatoes": 2809},
+    {"date": "Mar 26", "Apples": 3129, "Oranges": 1726, "Tomatoes": 2290},
+]
+
+barChart = dmc.BarChart(
+    id={"type": "chart", "index": "bar"},
+    h=300,
+    dataKey="date",
+    data=barchart_data,
+    withLegend=True,
+    type="stacked",
+    series=[
+        {"name": "Apples", "color": "indigo.6"},
+        {"name": "Oranges", "color": "blue.6"},
+        {"name": "Tomatoes", "color": "teal.6"},
+    ],
+    highlightHover=True
+)
+
+
+
+areaChart = dmc.AreaChart(
+    id={"type": "chart", "index": "area"},
+    h=300,
+    dataKey="month",
+    data=data,
+    series=[
+        {"name": "Smartphones", "color": "violet.6", "inactiveColor": "violet.1"},
+        {"name": "Laptops", "color": "blue.6", "inactiveColor": "blue.1"},
+        {"name": "Tablets", "color": "teal.6", "inactiveColor": "teal.1"},
+    ],
+    curveType="linear",
+    tickLine="xy",
+    withGradient=False,
+    withXAxis=False,
+    withDots=False,
+    withLegend=True,
+)
+
+lineChart = dmc.LineChart(
+    id={"type": "chart", "index": "line"},
+    h=300,
+    dataKey="month",
+    data=data,
+    withLegend=True,
+    series=[
+        {"name": "Smartphones", "color": "violet.6", "inactiveColor": "violet.1"},
+        {"name": "Laptops", "color": "blue.6", "inactiveColor": "blue.1"},
+        {"name": "Tablets", "color": "teal.6", "inactiveColor": "teal.1"},
+    ],
```

Comment:
Found something weird... if you set `withTooltip=False` on the lineChart, then the click/hoverSeriesName is either null or None

Reply:
Actually, when you specify `withTooltip=False`, the docs say that "It also removes the cursor line and disables interactions with the chart."   So that might be the issue?  If so, I could just note that in the docs.

https://mantine.dev/charts/line-chart/#remove-tooltip